cli/src/template/init/src/bin/ios.rs

```diff
@@ -1,5 +1,23 @@
+use mopro_ffi::app_config::ios::IOS_ARCHS;
+
 fn main() {
+    let ios_archs: Vec<String> = if let Ok(ios_archs) = std::env::var("IOS_ARCHS") {
+        ios_archs.split(',').map(|arch| arch.to_string()).collect()
+    } else {
+        // Default case: select all supported architectures if none are provided
+        IOS_ARCHS.iter().map(|&arch| arch.to_string()).collect()
+    };
+
+    // Check 'IOS_ARCH' input validation
+    for arch in &ios_archs {
+        assert!(
+            IOS_ARCHS.contains(&arch.as_str()),
+            "Unsupported architecture: {}",
+            arch
+        );
+    }
+
     // A simple wrapper around a build command provided by mopro.
     // In the future this will likely be published in the mopro crate itself.
-    mopro_ffi::app_config::ios::build();
+    mopro_ffi::app_config::ios::build(&ios_archs);
```

Comment:
I think passing `archs`  as inputs doesn't make much sense to me
and I think letting users maintaining the line 4-18 is too much work
(and we have to maintain double files in template and test-e2e, and also documentation)

if it is in `env`, it can just let `mopro_ffi::app_config::ios::build()` to read the `env` and handle the error assertions?
like the debug/release mode, it is also handled by `mopro_ffi::app_config::ios::build()`